cpp/include/cudf/column/column_device_view.cuh

```diff
@@ -14,32 +14,38 @@
  * limitations under the License.
  */
 #pragma once
+#ifndef CUDF_RUNTIME_JIT
```

Comment:
Does clang-format still order our includes correctly with this macro separating them into two blocks?

Reply:
Yes, it does